crates/swc_ecma_parser/src/parser/expr.rs

```diff
@@ -1880,7 +1899,10 @@ impl<I: Tokens> Parser<I> {
         }
 
         if is!(self, ';')
-            || (!is!(self, '*') && !cur!(self, false).map(Token::starts_expr).unwrap_or(true))
+            || (!is!(self, '*')
+                && !is!(self, '/')
+                && !is!(self, "/=")
+                && !cur!(self, false).map(Token::starts_expr).unwrap_or(true))
```

Comment:
Babel and esbuild has a little different logic here, I think we should refactor it, I will put in TODO too